### PR TITLE
manpage: reword confusing subs-forced-only description

### PR DESCRIPTION
**Pull request by rnpnr**

### CODE REVIEW

File: DOCS/man/options.rst [hunk 1]
```diff
@@ -2607,11 +2607,11 @@ Subtitles
...
 ``--sub-forced-only=<auto|yes|no>``
-    Display only forced subtitles for the DVD subtitle stream selected by e.g.
-    ``--slang`` (default: ``auto``). When set to ``auto``, enabled when the
-    ``--subs-with-matching-audio`` option is on and a non-forced stream is selected.
-    Enabling this will hide all subtitles in streams that don't make a distinction
-    between forced and unforced events within a stream.
+    Display only forced subtitles for the DVD subtitle stream selected by
+    e.g. ``--slang`` (default: ``auto``). When set to ``auto``, and the
+    ``--subs-with-matching-audio`` option is set, a non-forced stream will
+    be selected. Enabling this will hide all subtitles in streams that don't
```
Comment (sfan5) [inline DOCS/man/options.rst:2613]: I don't think this is correct. This doesn't influence stream selection, only display, right?

Comment (rnpnr) [reply DOCS/man/options.rst:2613]: I guess that was also a poor choice of words. A better way putting it is:

> a non-forced stream will be **displayed**.

**Comment by rnpnr** [comment]
Not sure why this was marked as "Stalled". But in any case I rebased on
current master and made the change I suggested.

**Comment by Dudemanguy** [comment]
Thanks for updating this, but this option gave me a lot of headaches today and actually I'm planning to change it here: #12279. Your description is certainly better than what's currently there at least (nit: it also applies to PGS not just DVD).

**Comment by rnpnr** [comment]
No worries. Feel free to close this if you are going forward with #12279. I
only opened this because I was trying to figure out what `sub-forced-only`
was doing in my config file and the manpage was little help. According
to git-blame I added it years ago with no explanation ¯\_ (ツ)_/¯

**Comment by Dudemanguy** [comment]
Superseded by https://github.com/mpv-player/mpv/pull/12279.

**Pull request closed without merging**